algebraic-graphs.cabal

```diff
@@ -103,13 +113,17 @@ test-suite test-alga
                         Algebra.Graph.Test.IntAdjacencyMap,
                         Algebra.Graph.Test.Relation
     build-depends:      algebraic-graphs,
-                        base       >= 4.9,
-                        containers >= 0.5,
+                        base,
+                        base-compat,
+                        base-orphans >= 0.6 && <0.7,
```

Comment:
I presume `base-compat` and `base-orphans` are small, standard and stable solutions, so depending on them is not a big maintenance burden?

Reply:
@RyanGlScott is one of maintainers of both libraries, they are well maintained.

Wouldn't say they are *standard* (there a bunch of alternative preludes out there), but they have quite well known packages as revdeps:
- http://packdeps.haskellers.com/reverse/base-orphans
- http://packdeps.haskellers.com/reverse/base-compat

Reply:
I see, thanks.